src/accelerate/test_utils/testing.py

```diff
@@ -65,6 +65,18 @@ def parse_flag_from_env(key, default=False):
     return _value
 
 
+def parameterized_custom_name_func(func, param_num, param):
+    # customize the test name generator function as we want both params to appear in the sub-test
+    # name, as by default it shows only the first param
+    param_based_name = parameterized.to_safe_name("_".join(str(x) for x in param.args))
+    return f"{func.__name__}_{param_based_name}"
+
+
+SAFETENSORS = True
+PYTORCH = False
+SAVE_TYPES = (SAFETENSORS, PYTORCH)
```

Comment:
Why use the generic name `SAVE_TYPES`? I think it would be easier to understand the code if this is just deleted and the test changed like this:

    @parameterized.expand([True, False], name_func=parameterized_custom_name_func)
    def test_save_load_model(self, use_safetensors):

It would make sense to have a `SAVE_TYPES` if there were more than two options, but it's unlikely that `use_safetensors` will ever take values other than `True` or `False`.

Reply:
`SAVE_TYPES`/this method of doing it lets the tests be reported as `test_save_load_model_safetensors` and `test_save_load_model_pytorch`. Since we use it in more than one file, brought it out here.

Otherwise it'd be reported as `test_save_load_model_False`

Reply:
Although that *is* what is actually happening here too, so this abstraction is unnecessary in the end, I'll remove it.